models/opensea/ethereum/opensea_ethereum_events.sql

```diff
@@ -36,11 +36,11 @@ FROM
                 platform_fee_amount_raw,
                 platform_fee_amount,
                 platform_fee_amount_usd,
-                platform_fee_percentage,
+                CAST(platform_fee_percentage AS DECIMAL(38,0)) platform_fee_percentage,
```

Comment:
Probably this and other casts of percentages should be to Double and not Decimal(38, 0).

Reply:
I'll fix this 👍